pylsp/_utils.py

```diff
@@ -14,7 +14,7 @@
 JEDI_VERSION = jedi.__version__
 
 # Eol chars accepted by the LSP protocol
-EOL_CHARS = ['\r\n', '\r', '\n']
+EOL_CHARS = ['\n', '\r\n', '\r']
```

Comment:
The ordering here was done for performance reasons.So, please revert this change.

Reply:
Seems like it's worth a comment in that case